README.md

```diff
@@ -93,14 +105,23 @@ Sonar check regions (sonar_regions) possible values:
 ```yaml
 ---
   octodns:
+    healthcheck:
+      frequency: 60
```

Comment:
@ross @viranch I would like to propose to add this common field and have the providers mapped this to a sensible value that is supported.

Reply:
No opposed to the idea, but the devil may be in the details/rollout. IF all providers can translate it into something meaningfully it'd be fairly straightforward to do, but would probably take a while to get all the coordinated PRs through. Until then we'd probably need to document it with a caveat that it's WIP and people will need to check the specifics of their provider(s)

Reply:
I would only advertise this "globally" when all providers have implemented that. Before that, each provider could document that this is available as an alternative.
Maybe we could set 2.0 as a milestone for that.